plugins/chat/config/locales/client.en.yml

```diff
@@ -101,6 +101,22 @@ en:
         cannot_see:
           one: "%{usernames} cannot access this channel and was not notified."
           other: "%{usernames} cannot access this channel and were not notified."
+        
+        groups:
+          header:
+            some: "Some users won't be notified"
+            all: "Nobody will be notified"
+          unreachable:
+            one: "@%{group} doesn't allow mentions"
+            other: "@%{group} and @%{group_2} doesn't allow mentions"
+          unreachable_multiple: "@%{group} and %{count} others doesn't allow mentions"
+          too_many_members:
```

Comment:
nit: if `%{limit}` is `1`, these should say `...%{limit} user` instead of `...%{limit} users`